src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
+        {
+
+            /// <summary>
+            /// Test Case 23466739: BVT: Standard/Scientific: Verify history flyout
+            /// This automated test verifies using using the history history flyout in Standard mode
+            /// </summary>
+
+            //Verify selecting history items form the history flyout, using up/down arrows and [enter] to select history items,
+            // and clearing the history flyout.  Additionally, verify history panel and history flyout have same information.
+
+            page.HistoryPanel.OpenHistoryPanel();
+
+            page.StandardOperators.NumberPad.Input(-3);
+            page.StandardOperators.PlusButton.Click();
+            page.StandardOperators.NumberPad.Input(-2.6);
+            page.StandardOperators.EqualButton.Click();
+
+            page.StandardOperators.NumberPad.Input(2);
+            page.StandardOperators.MinusButton.Click();
+            page.StandardOperators.NumberPad.Input(3);
+            page.StandardOperators.EqualButton.Click();
+
+            var historyPanelItems = page.HistoryPanel.GetAllHistoryListViewItems();
+            Assert.IsTrue(historyPanelItems[0].Text.Equals("2 Minus ( 3= Minus (1", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyPanelItems[1].Text.Equals("-3 + -2.6= Minus (5.6", StringComparison.InvariantCultureIgnoreCase));
+
+            Assert.AreEqual("-1", page.GetCalculatorResultText());
+
+            page.CalcHistoryFlyout.OpenHistoryFlyout();
+
+            var historyItems = page.CalcHistoryFlyout.GetAllHistoryFlyoutListViewItems();
+            Assert.IsTrue(historyItems[0].Text.Equals("2 Minus ( 3= Minus (1", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyItems[1].Text.Equals("-3 + -2.6= Minus (5.6", StringComparison.InvariantCultureIgnoreCase));
+
+            Assert.AreEqual("-1", page.GetCalculatorResultText());
+
+            page.CalcHistoryFlyout.ListViewItem.SendKeys(Keys.ArrowDown + Keys.Enter);
+
+            Assert.AreEqual("-5.6", page.GetCalculatorResultText());
+            Assert.AreEqual("-3 + -2.6=", page.GetCalculatorExpressionText());
+
+            page.CalcHistoryFlyout.OpenHistoryFlyout();
+            page.CalcHistoryFlyout.ListViewItem.SendKeys(Keys.ArrowDown + Keys.ArrowUp + Keys.Enter);
+            Assert.AreEqual("-1", page.GetCalculatorResultText());
+
+            page.CalcHistoryFlyout.OpenHistoryFlyout();
+            page.CalcHistoryFlyout.ClearHistoryButton.Click();
+
+            page.CalcHistoryFlyout.OpenHistoryFlyout();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryEmpty"));
+
+            page.HistoryPanel.OpenHistoryPanel();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryEmpty"));
+        }
+
+        [TestMethod]
+        [Priority(2)]
+        public void BVT_ArithmeticOperators()
```

Comment:
Aren't these already tested in the smoke tests? How is this test different from the other tests that already use the standard operators?